Doc/c-api/slice.rst

```diff
@@ -1,5 +1,16 @@
 .. highlightlang:: c
 
+Ellipsis Object
```

Comment:
Since the file is called `slice.rst` and slices are more used than Ellipsis I think Slice Objects should be before Ellipsis Object.